resources/views/components/cd/form/radios.blade.php

```diff
@@ -0,0 +1,24 @@
+<x-cd.form.form-item field="{{ $name ?? $attributes->whereStartsWith('wire:model')->first() }}" 
+        classes="{{ $classes ?? '' }}" 
+        required="{{ (($required??'') === 'false') ? 0 : (boolval($required??'')?1:0) }}"
+        :description="$description ?? ''"
+>
+@php 
+  $field=$name ?? $attributes->whereStartsWith('wire:model')->first();
+@endphp
+    <x-slot name="field_title">{!! $label !!}</x-slot>
+    <div class="flex-grid compact-rows no-margin">
+    @foreach ($radiobuttons as $rad) 
+        <div class="form-item">
```

Comment:
```suggestion
        <div class="form-item" @if($inline ?? false) style="flex-basis: fit-content" @endif>
```
Here's an idea for being able to have inline radio options.